branchout-maven

```diff
@@ -64,7 +64,7 @@ function mavenSettings() {
 <settings xsi:schemaLocation=\"http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd\" xmlns=\"http://maven.apache.org/SETTINGS/1.1.0\"
     xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\">
 
-  <localRepository>${BRANCHOUT_STATE}/maven/repository</localRepository>
+  <localRepository>\${user.home}/branchout/${BRANCHOUT_NAME}/maven/repository</localRepository>
```

Comment:
as per discussion, we should just use cygpath to get the value correcct on windows

Reply:
the problem is ultimately because branchout is unix based but the settings file is read from a native context i.e. windows

Reply:
Resolved and all checks passed, approve and if I can I'll push direct.